Sources/BrowserServicesKit/ContentScopeScript/ContentScopeUserScript.swift

```diff
@@ -27,12 +27,16 @@ public final class ContentScopeProperties: Encodable {
     public let globalPrivacyControlValue: Bool
     public let debug: Bool = false
     public let sessionKey: String
+    public let language: String
     public let platform = ContentScopePlatform()
     public let features: [String: ContentScopeFeature]
 
     public init(gpcEnabled: Bool, sessionKey: String, featureToggles: ContentScopeFeatureToggles) {
         self.globalPrivacyControlValue = gpcEnabled
         self.sessionKey = sessionKey
+        let locale = Locale.current.identifier
+        let underscore = locale.firstIndex(of: "_")!
+        self.language = String(locale[..<underscore])
```

Comment:
Replacing these lines with: 
`self.languageCode = Locale.current.languageCode ?? "en"` 
should work nicely for what you are doing here & will default to `en` if for some reason the system cannot determine the language code

Reply:
Oooh good call! We've got a default of `en` on the autofill side as well, but happy to have one on both.

From what I saw `Locale.current.languageCode` was returning `en-US` (the autofill codebase expects just the two-letter language `en`). Maybe I'm just seeing things/mis-remembering things! Either way, I can do the string splitting on that side to keep it simple here 🙂